### ISSUE TITLE
JF wont scan TV Shows

### ISSUE THREAD

**Issue by 1024mb** [jellyfin/jellyfin#3400]
**Describe the bug**

I have added two TV Show libraries to JF, both of those libraries have not been scanned completely. From the library 1, only two entries are shown, from library 2 only 4. Library 1 shows two entries of which only one shows its poster and all metadata and the other is blank. Library 2 shows all 4 shows with metadata OK. I have tried re-scanning and selecting "Scan for new and updated files" and also "Replace all metadata", both didn't gave any good results.  
Between both libraries there should be ~40 shows, not 6. I have set up JF to download all metadata for TV Shows (those are the only media I don't have local metadata for, except for the TV Shows info, local episodes metadata is missing so it have to be downloaded from the Internet along with the TV Show metadata as I don't expect JF to pick up first the local info and the search for the missing stuff).

**System (please complete the following information):**
 - OS: Windows 10
 - Virtualization: 
 - Clients: Browser
 - Browser: Firefox
 - Jellyfin Version: 10.6.0 nightly
 - Playback: 
 - Installed Plugins: Fanart, trakt (trakt is not being used)
 - Reverse Proxy: nope
 - Base URL: nope
 - Networking: 
 - Storage: local

**To Reproduce**

1. Create TV Show library
2. Add local paths
3. Set up to download all the data

**Expected behavior**

JF to scan the TV Shows successfully and show them on the library.

**Logs**

```
[10:21:56] [ERR] [37] MediaBrowser.Providers.Manager.ProviderManager: Error refreshing item
MediaBrowser.Model.Net.HttpException: Forbidden
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\HttpClientManager\HttpClientManager.cs:line 329
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\HttpClientManager\HttpClientManager.cs:line 266
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\HttpClientManager\HttpClientManager.cs:line 172
   at MediaBrowser.Providers.Manager.ProviderManager.SaveImage(BaseItem item, String url, ImageType type, Nullable`1 imageIndex, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 152
   at Emby.Server.Implementations.Library.LibraryManager.ConvertImageToLocal(BaseItem item, ItemImageInfo image, Int32 imageIndex) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 2892
   at Emby.Server.Implementations.Library.LibraryManager.UpdateImages(BaseItem item, Boolean forceUpdate) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 1883
   at Emby.Server.Implementations.Library.LibraryManager.UpdateItems(IEnumerable`1 items, BaseItem parent, ItemUpdateType updateReason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 1942
   at Emby.Server.Implementations.Library.LibraryManager.UpdateItem(BaseItem item, BaseItem parent, ItemUpdateType updateReason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 1985
   at MediaBrowser.Controller.Entities.BaseItem.UpdateToRepository(ItemUpdateType updateReason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\BaseItem.cs:line 2312
   at MediaBrowser.Providers.Manager.MetadataService`2.SavePeopleMetadata(List`1 people, LibraryOptions libraryOptions, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\MetadataService.cs:line 248
   at MediaBrowser.Providers.Manager.MetadataService`2.SaveItem(MetadataResult`1 result, LibraryOptions libraryOptions, ItemUpdateType reason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\MetadataService.cs:line 211
   at MediaBrowser.Providers.Manager.MetadataService`2.RefreshMetadata(BaseItem item, MetadataRefreshOptions refreshOptions, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\MetadataService.cs:line 189
   at MediaBrowser.Controller.Entities.BaseItem.RefreshMetadata(MetadataRefreshOptions options, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\BaseItem.cs:line 1412
   at MediaBrowser.Controller.Entities.Folder.RefreshAllMetadataForContainer(IMetadataContainer container, MetadataRefreshOptions refreshOptions, IProgress`1 progress, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 505
   at MediaBrowser.Controller.Entities.Folder.RefreshChildMetadata(BaseItem child, MetadataRefreshOptions refreshOptions, Boolean recursive, IProgress`1 progress, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 517
   at MediaBrowser.Controller.Entities.Folder.RefreshMetadataRecursive(List`1 children, MetadataRefreshOptions refreshOptions, Boolean recursive, IProgress`1 progress, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 487
   at MediaBrowser.Controller.Entities.Folder.ValidateChildrenInternal2(IProgress`1 progress, CancellationToken cancellationToken, Boolean recursive, Boolean refreshChildMetadata, MetadataRefreshOptions refreshOptions, IDirectoryService directoryService) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 460
   at MediaBrowser.Controller.Entities.Folder.ValidateChildrenInternal(IProgress`1 progress, CancellationToken cancellationToken, Boolean recursive, Boolean refreshChildMetadata, MetadataRefreshOptions refreshOptions, IDirectoryService directoryService) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 298
   at MediaBrowser.Providers.Manager.ProviderManager.RefreshCollectionFolderChildren(MetadataRefreshOptions options, CollectionFolder collectionFolder, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 1084
   at MediaBrowser.Providers.Manager.ProviderManager.RefreshItem(BaseItem item, MetadataRefreshOptions options, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 1067
   at MediaBrowser.Providers.Manager.ProviderManager.StartProcessingRefreshQueue() in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 1040
```

I also get a lot of this:

```
[10:51:55] [ERR] [41] MediaBrowser.Providers.Manager.ProviderManager: SeasonProvider failed in GetImageInfos for type Season
System.MissingMethodException: Method not found: 'System.String MediaBrowser.Model.Entities.ProviderIdsExtensions.GetProviderId(MediaBrowser.Model.Entities.IHasProviderIds, MediaBrowser.Model.Entities.MetadataProviders)'.
   at Jellyfin.Plugin.Fanart.Providers.SeasonProvider.GetImages(BaseItem item, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Jellyfin.Plugin.Fanart.Providers.SeasonProvider.GetImages(BaseItem item, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.ProviderManager.GetImages(BaseItem item, CancellationToken cancellationToken, IRemoteImageProvider provider, List`1 preferredLanguages, Nullable`1 type) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 236
```

**Additional context**

I think I have explained everything up there.  
Something that may be unrelated specifically to this issue is that if you have local metadata for the TV Shows (I'm specifically talking about the extrafanart folder) and the library is NOT set to store the images into the media folders and also while performing a re-scanning you select "Replace all metadata" and also to replace all images, JF will delete all the files inside the extrafanart folder and wont download any file in there so it will end up empty. This of course is not the expected behavior, if I have set up the library to not store any files on the media directory it shouldn't also delete any files from there. The expected behavior is JF to replace all images from its data directory not the media folder.

Oh, and also I have tried uninstalling the fanart plugin and I got the exact same error.

**Comment by crobibero** [comment]
What metadata providers do you have enabled for that library?

The Fanart errors are because the plugins haven't been updated to support 10.6 yet.

**Comment by 1024mb** [comment]
> 
> 
> What metadata providers do you have enabled for that library?
> 
> The Fanart errors are because the plugins haven't been updated to support 10.6 yet.

I have deleted the plugin and still got the same error.
I have the library set up like this:

![firefox_2020-06-21_11-15-52](https://user-images.githubusercontent.com/9301204/85229666-c5100980-b3b0-11ea-871b-c80683058791.png)
![firefox_2020-06-21_11-16-03](https://user-images.githubusercontent.com/9301204/85229672-c9d4bd80-b3b0-11ea-9ce2-ddd290b04303.png)
![firefox_2020-06-21_11-16-11](https://user-images.githubusercontent.com/9301204/85229677-cccfae00-b3b0-11ea-9e81-966da2bb0a34.png)
![firefox_2020-06-21_11-16-25](https://user-images.githubusercontent.com/9301204/85229678-ce997180-b3b0-11ea-8853-df5197317c7e.png)

**Comment by crobibero** [comment]
Can you please attach more from the log? Hard to tell what is happening just from the exception

**Comment by 1024mb** [comment]
Sure, but that is all I got. This is the entire log since I opened JF:
```
C:\Programas\Jellyfin\Server Nigthly>jellyfin.exe -d C:\ProgramData\jellyfin\server\
[10:55:08] [INF] [1] Main: Jellyfin version: 10.6.0
[10:55:08] [INF] [1] Main: Environment Variables: ["[DOTNET_CLI_TELEMETRY_OPTOUT, 1]", "[JELLYFIN_LOG_DIR, C:\\ProgramData\\jellyfin\\server\\log]"]
[10:55:08] [INF] [1] Main: Arguments: ["C:\\Programas\\Jellyfin\\Server Nigthly\\jellyfin.dll", "-d", "C:\\ProgramData\\jellyfin\\server\\"]
[10:55:08] [INF] [1] Main: Operating system: Windows
[10:55:08] [INF] [1] Main: Architecture: X64
[10:55:08] [INF] [1] Main: 64-Bit Process: True
[10:55:08] [INF] [1] Main: User Interactive: True
[10:55:08] [INF] [1] Main: Processor count: 4
[10:55:08] [INF] [1] Main: Program data path: C:\ProgramData\jellyfin\server\
[10:55:08] [INF] [1] Main: Web resources path: C:\Programas\Jellyfin\Server Nigthly\jellyfin-web
[10:55:08] [INF] [1] Main: Application directory: C:\Programas\Jellyfin\Server Nigthly\
[10:55:08] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: C:\ProgramData\jellyfin\server\cache
[10:55:08] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[10:55:08] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly Trakt, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null from C:\ProgramData\jellyfin\server\plugins\Trakt\Trakt.dll
[10:55:09] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'AccessSchedule' is configured to use schema 'jellyfin'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[10:55:09] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'ActivityLog' is configured to use schema 'jellyfin'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[10:55:09] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'ImageInfo' is configured to use schema 'jellyfin'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[10:55:09] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'Permission' is configured to use schema 'jellyfin'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[10:55:09] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'Preference' is configured to use schema 'jellyfin'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[10:55:09] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'User' is configured to use schema 'jellyfin'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[10:55:11] [INF] [1] Main: Kestrel listening on all interfaces
[10:55:11] [INF] [1] Emby.Server.Implementations.ApplicationHost: Running startup tasks
[10:55:11] [INF] [1] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for Extraer imágenes de los capítulos set to fire at 06/22/2020 02:00, which is 15:04:48.1279358 from now.
[10:55:12] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version unknown
[10:55:12] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: Custom: Failed version check: C:\Programas (2)\ffmpeg\ffmpeg.exe
[10:55:12] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available decoders: ["h264", "h264_qsv", "h264_cuvid", "hevc", "hevc_qsv", "hevc_cuvid", "mpeg2video", "mpeg2_qsv", "mpeg2_cuvid", "mpeg4", "mpeg4_cuvid", "msmpeg4", "vc1_qsv", "vc1_cuvid", "vp8_cuvid", "vp8_qsv", "vp9_cuvid", "vp9_qsv", "aac", "ac3", "mp3"]
[10:55:12] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available encoders: ["libx264", "h264_amf", "h264_nvenc", "h264_qsv", "libx265", "hevc_amf", "hevc_nvenc", "hevc_qsv", "mpeg4", "msmpeg4", "libvpx", "libvpx-vp9", "aac", "ac3", "libmp3lame", "libopus", "libvorbis", "srt"]
[10:55:12] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available hwaccel types: ["cuda", "dxva2", "qsv", "d3d11va"]
[10:55:12] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: Custom: C:\Programas (2)\ffmpeg\ffmpeg.exe
[10:55:12] [INF] [1] Emby.Server.Implementations.ApplicationHost: ServerId: 9d906cebe8c94861a19c77d2e73f470c
[10:55:12] [INF] [13] Emby.Dlna.Main.DlnaEntryPoint: Registering publisher for urn:schemas-upnp-org:device:MediaServer:1 on 192.168.191.80
[10:55:12] [INF] [13] Emby.Dlna.Main.DlnaEntryPoint: Registering publisher for urn:schemas-upnp-org:device:MediaServer:1 on 192.168.100.155
[10:55:13] [INF] [13] Emby.Server.Implementations.ApplicationHost: Executed all pre-startup entry points in 0:00:00.4701359
[10:55:13] [INF] [13] Emby.Server.Implementations.ApplicationHost: Core startup complete
[10:55:13] [INF] [13] Emby.Server.Implementations.EntryPoints.ExternalPortForwarding: Starting NAT discovery
[10:55:13] [INF] [13] Emby.Server.Implementations.ApplicationHost: Executed all post-startup entry points in 0:00:00.610277
[10:55:13] [INF] [13] Main: Startup complete 0:00:05.7402341
[10:55:14] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: StartupTrigger fired for task: Actualizar complementos
[10:55:14] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queueing task PluginUpdateTask
[10:55:14] [INF] [14] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing Actualizar complementos
[10:55:14] [ERR] [14] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request: A task was canceled. URL: http://localhost:8096/web/touchicon.png
[10:55:15] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: Actualizar complementos Completed after 0 minute(s) and 0 seconds
[10:55:16] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[10:55:16] [WRN] [9] Emby.Server.Implementations.HttpServer.HttpListenerHost: HTTP Response 200 to 192.168.100.155. Time (slow): 0:00:00.5111982. http://192.168.100.155:8096/ScheduledTasks?IsEnabled=true
[10:55:18] [INF] [13] Emby.Dlna.Main.DlnaEntryPoint: DLNA Session created for [TV]Samsung LED40 - UN40F6400
[10:56:50] [INF] [14] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:192.168.100.155 request
[10:56:52] [INF] [24] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:192.168.100.155 closed
[10:56:56] [INF] [14] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS ::ffff:192.168.100.155 request
[10:57:45] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 76a377525881e731e98f289b944196e4
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 36de38bc5c0510bae06f07d68ce33ea7
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 79f95632409cdd7e62098b1140a94f8f
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 79f95632409cdd7e62098b1140a94f8f
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 6c550bb79e1d5b7ccc47b26ebeb46657
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 6c550bb79e1d5b7ccc47b26ebeb46657
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 36de38bc5c0510bae06f07d68ce33ea7
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart f0fec88d7fe3154e490ede75faa9ccad
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart b0baf5193e628a51da8d96f58ba0b235
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete b0baf5193e628a51da8d96f58ba0b235
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 8359255e567d6de2e50134e7d841cc08
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 8359255e567d6de2e50134e7d841cc08
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete f0fec88d7fe3154e490ede75faa9ccad
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart b8b1efaed3019f4e5897d6c764e33ffc
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 8315343313920dfc84e8768be054d3e5
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 8315343313920dfc84e8768be054d3e5
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 5269228d14ecedf6547c46996719c680
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 5269228d14ecedf6547c46996719c680
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 0c9b860a23664c372c6026a81c01745f
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 0c9b860a23664c372c6026a81c01745f
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart de8ec1f9dde5e44073a191f848d7e469
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete de8ec1f9dde5e44073a191f848d7e469
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart a617211853f372ae91cc3a04aeebd491
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete a617211853f372ae91cc3a04aeebd491
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 3f6721b80edb1f098c5c79de176682f1
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 3f6721b80edb1f098c5c79de176682f1
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 7bde31c22323c4f3fb0f74d0c5de4d35
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 7bde31c22323c4f3fb0f74d0c5de4d35
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 615370b532873482765984ebc8510540
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 615370b532873482765984ebc8510540
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart f1f359cadbf04f9827803620bcb8a104
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete f1f359cadbf04f9827803620bcb8a104
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 4250ba93b1489347c5daab6a9fd0001f
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 4250ba93b1489347c5daab6a9fd0001f
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete b8b1efaed3019f4e5897d6c764e33ffc
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 06fbfc99668b22e529e4c9841111fec8
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 127ad60d26c88dff1f67174163d66b64
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 127ad60d26c88dff1f67174163d66b64
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 06fbfc99668b22e529e4c9841111fec8
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart cfa06a3e1b6cf1c8ff8b5d70df4ff3b8
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart 6b8da3073da91847a7228718fe8d8613
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 6b8da3073da91847a7228718fe8d8613
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart b9b0f4370c159a5f752670286a096489
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete b9b0f4370c159a5f752670286a096489
[10:57:46] [INF] [24] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete cfa06a3e1b6cf1c8ff8b5d70df4ff3b8
[10:57:51] [INF] [12] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete 76a377525881e731e98f289b944196e4
[10:57:51] [ERR] [12] MediaBrowser.Providers.Manager.ProviderManager: Error refreshing item
MediaBrowser.Model.Net.HttpException: Forbidden
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\HttpClientManager\HttpClientManager.cs:line 329
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\HttpClientManager\HttpClientManager.cs:line 266
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\HttpClientManager\HttpClientManager.cs:line 172
   at MediaBrowser.Providers.Manager.ProviderManager.SaveImage(BaseItem item, String url, ImageType type, Nullable`1 imageIndex, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 152
   at Emby.Server.Implementations.Library.LibraryManager.ConvertImageToLocal(BaseItem item, ItemImageInfo image, Int32 imageIndex) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 2892
   at Emby.Server.Implementations.Library.LibraryManager.UpdateImages(BaseItem item, Boolean forceUpdate) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 1883
   at Emby.Server.Implementations.Library.LibraryManager.UpdateItems(IEnumerable`1 items, BaseItem parent, ItemUpdateType updateReason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 1942
   at Emby.Server.Implementations.Library.LibraryManager.UpdateItem(BaseItem item, BaseItem parent, ItemUpdateType updateReason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\Emby.Server.Implementations\Library\LibraryManager.cs:line 1985
   at MediaBrowser.Controller.Entities.BaseItem.UpdateToRepository(ItemUpdateType updateReason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\BaseItem.cs:line 2312
   at MediaBrowser.Providers.Manager.MetadataService`2.SavePeopleMetadata(List`1 people, LibraryOptions libraryOptions, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\MetadataService.cs:line 248
   at MediaBrowser.Providers.Manager.MetadataService`2.SaveItem(MetadataResult`1 result, LibraryOptions libraryOptions, ItemUpdateType reason, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\MetadataService.cs:line 211
   at MediaBrowser.Providers.Manager.MetadataService`2.RefreshMetadata(BaseItem item, MetadataRefreshOptions refreshOptions, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\MetadataService.cs:line 189
   at MediaBrowser.Controller.Entities.BaseItem.RefreshMetadata(MetadataRefreshOptions options, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\BaseItem.cs:line 1412
   at MediaBrowser.Controller.Entities.Folder.RefreshAllMetadataForContainer(IMetadataContainer container, MetadataRefreshOptions refreshOptions, IProgress`1 progress, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 505
   at MediaBrowser.Controller.Entities.Folder.RefreshChildMetadata(BaseItem child, MetadataRefreshOptions refreshOptions, Boolean recursive, IProgress`1 progress, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 517
   at MediaBrowser.Controller.Entities.Folder.RefreshMetadataRecursive(List`1 children, MetadataRefreshOptions refreshOptions, Boolean recursive, IProgress`1 progress, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 487
   at MediaBrowser.Controller.Entities.Folder.ValidateChildrenInternal2(IProgress`1 progress, CancellationToken cancellationToken, Boolean recursive, Boolean refreshChildMetadata, MetadataRefreshOptions refreshOptions, IDirectoryService directoryService) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 460
   at MediaBrowser.Controller.Entities.Folder.ValidateChildrenInternal(IProgress`1 progress, CancellationToken cancellationToken, Boolean recursive, Boolean refreshChildMetadata, MetadataRefreshOptions refreshOptions, IDirectoryService directoryService) in C:\Users\1024mb\jellyfin\MediaBrowser.Controller\Entities\Folder.cs:line 298
   at MediaBrowser.Providers.Manager.ProviderManager.RefreshCollectionFolderChildren(MetadataRefreshOptions options, CollectionFolder collectionFolder, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 1084
   at MediaBrowser.Providers.Manager.ProviderManager.RefreshItem(BaseItem item, MetadataRefreshOptions options, CancellationToken cancellationToken) in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 1067
   at MediaBrowser.Providers.Manager.ProviderManager.StartProcessingRefreshQueue() in C:\Users\1024mb\jellyfin\MediaBrowser.Providers\Manager\ProviderManager.cs:line 1040
```

**Comment by nextlooper42** [comment]
"Replace all metadata" option is not working at all btw. If you have incomplete or outdated metadata, "Replace all metadata" will do literally nothing... just keep the same stuff.

Also "Search for missing metadata" does nothing at all. I added new stuff for some episodes to TMDB, then used "Search for missing metadata" in Jellyfin for updating empty space in tv show (it should use fallback language, but thats broken af too) ... nothing new.

**Comment by Bond-009** [comment]
The `MediaBrowser.Model.Net.HttpException: Forbidden` exception is fixed now (#3409)
